Python/Product/Analysis/LanguageServer/Server.cs

```diff
@@ -153,6 +157,9 @@ private sealed class ReloadModulesQueueItem : IAnalyzable {
                 }
             };
         }
+
+        internal Task CompleteInitialization { get; private set; } = Task.CompletedTask;
```

Comment:
OK

Reply:
Although point here is NOT to await on DoInitializeAsync and just let it run. Awaiting blocks VSC startup.

Reply:
Actually, nothing ever called until `initialized` returns. So I don't think the task can be replaced easily.

Reply:
So do I get it right that you want to wait for initialization to complete on the first request after initialization, rather than during initialization request itself?

Reply:
Yep. Those are 'open file' and 'change config' - they come after the init. Actually we need to debug why sync context does not wait for completions - or, rather, the StreamJsonRpc.

Reply:
I might need to use alternative implementation (from R/vsc) instead of sync context which actually makes sure requests do complete sequentially. See https://github.com/MikhailArkhipov/PTVS/blob/sync/Python/Product/VSCode/AnalysisVsc/LanguageServerProxy.cs